helm/kmcp/templates/rbac/clusterrole.yaml

```diff
@@ -11,6 +11,7 @@ rules:
   resources:
   - configmaps
   - services
+  - secrets
```

Comment:
We should NOT have and write roles for secrets: e.g. (Create/update/delete/patch). Please switch these to a read-only role